sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/ProcessSessionMessageEventArgs.cs

```diff
@@ -49,6 +49,16 @@ public class ProcessSessionMessageEventArgs : EventArgs
         /// </summary>
         public DateTimeOffset SessionLockedUntil => _sessionReceiver.SessionLockedUntil;
 
+        /// <summary>
+        /// The entity path corresponding to the entity that the message was received from.
```

Comment:
```suggestion
        /// The path of the Service Bus entity that the message was received from.
```

Based on the [Receiver comment](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/servicebus/Azure.Messaging.ServiceBus/src/Receiver/ServiceBusReceiver.cs#L34)